### PR TITLE
Add missing space in `RuboCop::Cop::Style::MultipleComparison::MSG`

### LINKED ISSUE #4422: The Style/MultipleComparison cop's message is missing a space between words
The message currently reads:

```
Avoid comparing a variable with multiple itemsin a conditional, use Array#include? instead.
```

But it should read:

```
Avoid comparing a variable with multiple items in a conditional, use Array#include? instead.
```

### PR DESCRIPTION
**Pull request by timrogers**
The message currently reads:

```
Avoid comparing a variable with multiple itemsin a conditional, use Array#include? instead.
```

But it will now read:

```
Avoid comparing a variable with multiple items in a conditional, use Array#include? instead.
```

Fixes #4422.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### Bug fixes
 
+* [#4422](https://github.com/bbatsov/rubocop/issues/4422): Fix missing space in message for `Style/MultipleComparison`. ([@timrogers][])
```
Comment (bbatsov) [inline CHANGELOG.md]: You should add yourself to the bottom of the changelog.

**Comment by timrogers** [comment]
Sure - will get this updated tonight.

**Comment by timrogers** [comment]
@bbatsov Fixed!

**Comment by bbatsov** [comment]
I don't see any change.

**Comment by timrogers** [comment]
@bbatsov Oops! It's there now.

**Pull request merged**